istio/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
```

Comment:
I think @lluunn do most of the ISTIo work so I would list him and you as opposed to @kkasravi as an explicit approver.

Reply:
Made this change.